### PR TITLE
Update README.md for release v0.6.0

### LINKED ISSUE #189: Preparation for release
#### Motivation / Context

A version including the following changes is planned to be released:

1. Improved error handling (#132).
2. Structured logging (#133).
3. Perun node TUI
4. Support for ERC20 tokens.

In preparation for this release, also
1. Update the versions of go-perun, go and golangci-lint versions used in the CI setup.
2. Include TUI build in `make` command.

#### Testing

CI pipeline should pass after the upgrade.

#### Thread Safety

NA

### PR DESCRIPTION
**Pull request by manoranjith**
#### Description

- Include currency as a feature.
- Include links to video tutorial on perunnodetui app.

##### Category

Implementation Task

##### Relevant issue

Resolves 189.

##### Steps to run the tests

1. 
2. 
3.  

#### Checklist 

- [x] Name is added to the NOTICE file, if it is not present already.
- [x] Changes are rebased onto the target branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,12 +68,20 @@ This project currently contains two executable packages located in the `cmd` dir
...
+  visual representation most of the actions, it is easy way to try out payment
+  channel API and understand the perun protocol.
+
+For detailed information on using perun-node with `perunnodecli`, see the
 [tutorial section](https://labs.hyperledger.org/perun-doc/node/introduction.html#user-guide)
-on the project documentation website. For a tutorial using for `perunnodetui`, see
+on the project documentation website.
+
+For detailed information on using perun-node with `perunnodetui`, see the
 [this section](cmd/perunnodetui/Readme.md).
```
Comment (danielksan81) [inline README.md]: Does it make sense  to include the link already in the first part like in the perunnodecli part directly above?

Comment (manoranjith) [reply README.md]: I have combined the two lines and also rebased to latest develop.

File: README.md [hunk 2]
```diff
@@ -67,11 +68,18 @@ This project currently contains two executable packages located in the `cmd` dir
...
+  channel API and understand the perun protocol.
+
+For a tutorial on using perun-node with `perunnodecli`, see the
 [tutorial section](https://labs.hyperledger.org/perun-doc/node/introduction.html#user-guide)
-on the project documentation website. For a tutorial using for `perunnodetui`, see
-[this section](cmd/perunnodetui/Readme.md).
+on the project documentation website.
+
+For a tutorial on using perun-node with `perunnodetui`, see the
+[this section](cmd/perunnodetui/Readme.md). A talk on overview of perun,
```
Comment (danielksan81) [inline README.md]: see [this section], remove the "the" in the sentence, then it seems to be fine :)

Comment (manoranjith) [reply README.md]: I have updated.

**Review by danielksan81 — APPROVED** [review]
ok

**Pull request merged**